polaris.shopify.com/src/components/TokenList/TokenList.tsx

```diff
@@ -446,6 +446,9 @@ function TokenPreview({name, value}: TokenPreviewProps) {
         {...previewDivAttributes}
         style={{
           display: 'flex',
+          background: 'var(--p-color-bg-app)',
+          borderRadius: 'var(--p-border-radius-200)',
+          padding: '1rem',
         }}
       >
         <div
```

Comment:
For a slightly nicer layout, it would be great to have the inner box stretch to full width of its container:

<img width="586" alt="Screenshot 2023-10-13 at 12 02 18 pm" src="https://github.com/Shopify/polaris/assets/612020/58ac8c5f-a131-4ecf-a145-e2d499eec051">

Can you make the following change to this `div` to enable it:

```diff
 <div
   style={{
-    width: 50,
+    width: '100%',
+    minWidth: 50,
     minHeight: 50,
```